Les sous-répertoires pugixml-1.2.orig/contrib et pugixml-1.2/contrib sont identiques
Les sous-répertoires pugixml-1.2.orig/docs et pugixml-1.2/docs sont identiques
diff -Nu pugixml-1.2.orig/meson.build pugixml-1.2/meson.build
--- pugixml-1.2.orig/meson.build	1970-01-01 01:00:00.000000000 +0100
+++ pugixml-1.2/meson.build	2016-12-09 16:40:24.749898766 +0100
@@ -0,0 +1,13 @@
+project('pugixml', 'cpp')
+
+install_headers(['src/pugixml.hpp', 'src/pugiconfig.hpp'])
+pugixml = library('pugixml',
+                  ['src/pugixml.cpp'],
+                  install:true)
+
+pkg_mod = import('pkgconfig')
+pkg_mod.generate(libraries : pugixml,
+                 version : '1.2',
+                 name : 'libpugixml',
+                 description : 'pugixml',
+                 filebase : 'pugixml')
\ Pas de fin de ligne à la fin du fichier
Les sous-répertoires pugixml-1.2.orig/scripts et pugixml-1.2/scripts sont identiques
Les sous-répertoires pugixml-1.2.orig/src et pugixml-1.2/src sont identiques
